templates/bug/_list.html.twig

```diff
@@ -1,6 +1,6 @@
 <div class="row" {{ stimulus_target('search', 'container') }}>
     {% for bug in bugs %}
-        {{ include('bug/_card.html.twig', { bug }) }}
+        {{ include('bug/_card.html.twig', {bug}) }}
```

Comment:
Qu'estr-ce que c'est que ce for else ?

Reply:
https://twig.symfony.com/doc/3.x/tags/for.html#the-else-clause